src/actions/markdownActions.js

```diff
@@ -0,0 +1,17 @@
+
+
+export const SAVE_MARKDOWN = 'SAVE_MARKDOWN';
```

Comment:
this card was only about refactoring to use flux. these actions add functionality